File: CTDK_Python/098/_098.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
i = 2
while(i <= n):
    s = (i + s)**(1 / i)
    i = i + 1

print("Tong can la: ", s)

File: LTHAM/098/098.cpp
#include <iostream>
#include <cmath>
using namespace std;

float Tong(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	
	cout << "S = " << Tong(n) << endl;
	return 0;
}

float Tong(int n)
{
	float s = 0;
	int i = 2;
	while (i <= n)
	{
		s = pow(s + i, (float)1 / i);
		i++;
	}
	return s;
}